nuxeo-core/nuxeo-core/src/main/java/org/nuxeo/ecm/core/api/AbstractSession.java

```diff
@@ -1584,7 +1584,7 @@ public DocumentModel saveDocument(DocumentModel docModel) {
             // - previous document is checkouted
```

Comment:
Don't you want this commment? https://github.com/nuxeo/nuxeo-lts/pull/252/files#diff-99033aaf23b105e521257ea5a0f7bd523f6302a16d04ed0ac31558eeb9be79c4R1651

Reply:
yes :+1: